msal/src/androidTest/java/com/microsoft/identity/client/LoggerTest.java

```diff
@@ -285,11 +282,15 @@ private void verifyLogMessage(final RequestContext requestContext, final LogResp
 
     private void verifyLogMessageFormat(final LogResponse response, final String message, final RequestContext requestContext,
                                         final Throwable throwable) {
-        Assert.assertTrue(response.getMessage().contains("MSAL " + PublicClientApplication.getSdkVersion() + " Android " + Build.VERSION.SDK_INT + " ["));
+        Assert.assertTrue(response.getMessage().contains(" SDK ver:" + VERSION_NAME + " Android " + Build.VERSION.SDK_INT));
         if (requestContext != null && (requestContext.getCorrelationId() != null || !MsalUtils.isEmpty(requestContext.getComponent()))) {
             if (requestContext.getCorrelationId() != null && !MsalUtils.isEmpty(requestContext.getComponent())) {
-                Assert.assertTrue(response.getMessage().contains(" - " + requestContext.getCorrelationId().toString() + "] (" + requestContext.getComponent()
-                        + ") " + message + getStackTrace(throwable)));
+                final String expectedStr = " - " + requestContext.getCorrelationId().toString()
```

Comment:
I'm expecting this test to break when I merge in my latest logging changes. I'll take care of modifying this test when the time comes